packages/fluentui/docs/src/prototypes/VirtualizedStickyTree/itemsGenerator.ts

```diff
@@ -0,0 +1,24 @@
+import * as _ from 'lodash';
+
+function getItems(minItems = 5, maxItems = 15, maxLevel = 2, maxSticky = 10) {
```

Comment:
Propose add types in this file. 
 @YuanboXue-Amber , I apologize in advance for such a question, I have not found the answer anywhere, is the `fluentui/packages/fluentui/docs/src/prototypes/` a folder of experimental components?